web/src/components/CompareSide.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <div>
+    <component :is="'style'" type="text/css">
+      .marked-code, .token.marked-code {
+        background: var(--markedbg);
+        text-shadow: none;
+      }
+      <template v-for="item in hoveringSelections">
+        .token.operator.{{ item }}, .token.entity.{{ item }}, .token.url.{{ item }},
+        .language-css .token.string.{{ item }}, .style .token.string.{{ item }}, .token.marked-code.{{ item }} {
```

Comment:
Why not just use `.{{ item }}` here? Which classes you you want to avoid?

Reply:
prism add a special background to punctuation and characters like `||`. To overwrite this a more specific css selector is needed. These long selectors can be avoided by using `!important` but that is considered bad style.